lib/adt.ex

```diff
@@ -57,7 +54,7 @@ defmodule ADT do
 
   # Maps a module like Foo.Bar.Baz into a short string "Baz"
   def _shorten(name) do
-    Regex.named_captures(~r/\.(?<short>[^\.{]+)($|{)/, inspect(name), include_captures: true) |> Map.fetch!("short")
+    Regex.named_captures(~r/.(?<short>[^.{]+)($|{)/, inspect(name), include_captures: true) |> Map.fetch!("short")
```

Comment:
Do you think it's best to interpolate the terminator so that it becomes a simpler regex? Or maybe two regexes was ok